moler/events/lineevent.py

```diff
@@ -13,21 +13,19 @@
 
 
 class LineEvent(TextualEvent):
-    def __init__(self, detect_pattern=None, detect_patterns=list(), connection=None, till_occurs_times=-1):
+    def __init__(self, detect_patterns=list(), connection=None, till_occurs_times=-1, match='any'):
```

Comment:
Whad do you think about  detect_patterns without default value?

Reply:
Where is runner?

Reply:
Good point. detect_patterns should be required parameter. I added runner for all events